bokeh/document/document.py

```diff
@@ -379,19 +350,7 @@ def add_timeout_callback(self, callback: Callback, timeout_milliseconds: int) ->
         '''
         from ..server.callbacks import TimeoutCallback
         cb = TimeoutCallback(callback=None, timeout=timeout_milliseconds, callback_id=make_id())
-        return self._add_session_callback(cb, callback, one_shot=True)
-
-    def apply_json_event(self, json: EventJson) -> None:
```

Comment:
What happened here? This was public API and its removal broke `jupyter_bokeh`.

Reply:
> What happened here? 

There are no tests anywhere that exercise Bokeh and `jupyter_bokeh` together, AFAIK.

Reply:
Agreed, need to fix that, I'll see if I can put some basic tests together (and ideally remove any usage of private APIs from `jupyter_bokeh`).

Reply:
Ideally we'd have had a deprecation cycle for the removal of this public API though.

Reply:
Just to mention, for several of these functions I did actually try to do some due diligence and search all of GitHub for any usage, but all I ever saw were main repo clones. Evidently I missed this, though.